database/src/main/postgres/runs/V1.5.11__get_partitioning_measures.sql

```diff
@@ -63,12 +63,6 @@ BEGIN
     FROM runs.measure_definitions AS md
     WHERE md.fk_partitioning = _fk_partitioning;
 
```

Comment:
Unfortunately, _FlyWay_ doesn't work this way. You have to create a new version of the function under a new minor number (am I right @salamonpavel ?_. I believe we are at 1.16.x now.